Maestro/subscriptions.json

```diff
@@ -184,6 +198,20 @@
             "-GitHubPullRequestNotifications 'dotnet/corefx-contrib'",
             "-DirPropsVersionElements CoreClrExpectedPrerelease"
           ]
+        },
+        // This handler will bring the Latest CoreCLR master build into CoreCLR master
```

Comment:
This lets CoreCLR build tests based on an existing CoreCLR package, rather than only the just-built CoreCLR. It's purely a test dependency. (The CoreFX auto-upgrade is also purely for tests.)

Might be useful to clarify in an extended build-info--whether a certain dependency is a product dependency or a test/utility dependency.

---
In reply to: [71458883](https://github.com/dotnet/versions/pull/40#discussion_r71458883) [](ancestors = 71458883)